HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # History
 
+## 0.6.5 - Breaking changes
+
+- Return only single job from `Parser.parse` if `multiple_jobs` is set to False
```

Comment:
Vi skal huske at tage højde for denne ændring et par steder i https://github.com/peleiden/daLUKE